content/docs/rosa/gpu/_index.md

```diff
@@ -71,7 +69,7 @@ If you plan to use Helm to deploy the GPU operator, you will need to do the foll
    > The guide uses *g5.xlarge* as an example. Please be mindful of the GPU cost of the type you choose.
 
    ```bash
-   export GPU_INSTANCE_TYPE='g5.xlarge'
+   export GPU_INSTANCE_TYPE='g4dn.2xlarge'
```

Comment:
I do not think `g4dn` is newer.  According to [AWS](https://aws.amazon.com/ec2/instance-types/g5/), "G5 instances deliver up to 3x higher graphics performance and up to 40% better price performance than G4dn instances".  

Also - GPU instances are super $$$~ so let's stick to `xlarge` if we can.

Reply:
sure thing! I'll change this to stick with the xlarge then. Thanks for the tip!